library/src/main/resources/templatemaster/formdef/list-pager.html

```diff
@@ -1,28 +1,16 @@
 <div class="tm-list-pager text-right">
-    
     <span style="margin-right: 1em">
-    <form id="pageSize" method="get" action="${subUrl}">
-    	Page Size
-    	<select id="pageSize" name="p" >
-	    	[#assign pageSizeArray = [20,50,100,200]]
-	
-			[#list pageSizeArray as pageSize]
-			  <option value='${pageSize}' [#if currentUserPageSize == pageSize]selected[/#if]>${pageSize}</option>
-			[/#list]
-    	</select><br/>
-    </form>
-    	Page ${page.number + 1} of ${page.totalPages} &mdash; ${page.totalElements} Records
+    	1 - ${currentUserPageSize} of ${page.totalElements}&nbsp;
```

Comment:
@gvjoshi25 is this correct?  It seems to be missing parts, including the real start number (instead of always 1).

Reply:
Yes, code updated.